### ISSUE TITLE
Реквизиты для donation

### ISSUE THREAD

**Issue by motienko** [Olga-Yakovleva/RHVoice#75]
Ольга, есть ли возможность отправлять вам donation?

Например, в рублях или же какие-то альтернативные варианты (Litecoin/Bitcoin и т.п.) ?

**Comment by Olga-Yakovleva** [comment]
Это вопрос пока не решённый. Я ещё не пыталась выяснить, как это можно 
было бы правильно с точки зрения закона организовать и оформить, если 
это вообще возможно, учитывая, что синтезатор разрабатывает не 
какая-нибудь официальная организация, а один разработчик как частное лицо.

**Comment by CHERTS** [comment]
Ольга, в соответствии с Гражданским кодексом Российской Федерации: Статья 582, Вы имеете право принимать подертвования и они не облагаются налогом на прибыль.
Вы можете принимать подертвования любым законным в РФ способом (банковский перевод, электронные деньги и т.п.)

**Comment by Menelion** [comment]
Оля, в любых известных мне странах donations можно принимать без налогов и бюрократических проволочек. Как в России не знаю, а так можно:

1. Поставить кнопку PayPal;
2. Зарегистрироваться на Patreon. Минус - это сервис для регулярных пожертвований, поэтому желательно что-то регулярно делать... ну например, более или менее часто выпускать новые версии :).
3. У Яндекса есть какая-то благодарилка, но поскольку у нас он уже достаточно давно недоступен, ссылок не дам, к сожалению.
4. Можно просто в Readme или на Вики указать кошельки Webmoney, Я.денег или Киви - говорят, это тоже работает.

**Comment by Olga-Yakovleva** [comment]
Да, как вижу, согласно той самой статье 582, физическое лицо 
действительно имеет право принимать пожертвования без каких-либо 
дополнительных согласований. Конкретно к пожертвованиям, в отличии от 
подарков в целом, предъявляется требование использования их получателем 
для конкретных указанных целей. Интересно, как предполагается это 
интерпретировать в случае разработки частным лицом программы с открытыми 
исходниками. Хотя у моего синтезатора есть такие возможные прямые 
расходы: могло бы помочь платить за аренду сервера, на котором 
собираются голоса.

Нужно посмотреть, через какой сервис это лучше организовать, чтобы и 
русскоязычным пользователям было удобно воспользоваться, и приём 
пожертвований физическими лицами поддерживался. Последнему требованию, 
например, как я понимаю, тот же PayPal не удовлетворяет, так как они 
поддерживают только пожертвования организациям (наверное, для 
предотвращения возможного злоупотребления мошенниками).

**Comment by CHERTS** [comment]
На Paypal как раз основной контингент пожертвований - это физ. лица. То есть сбор пожертвований для ФЛ и от ФЛ.
Как ФЛ вы можете легко и беспроцентно выводить собранные пожертвования на счет в Российском банке. Я это делаю уже много лет и никаких проблем со стороны надзорных органов нет, никакие декларации и отчеты не сдаю.

**Comment by motienko** [comment]
для России удобно сервис от Яндекса.
https://yasobe.ru/

ну а paypal - для иностранных.

**Comment by CHERTS** [comment]
> для России удобно сервис от Яндекса.
> https://yasobe.ru/

И чем же он удобен? Для совершения пожертвования нужно иметь аккаунт на Yandex, а если его нет или он просто не нужен, то приехали.
У Paypal не обязательно иметь аккаунт, создаем кнопку для пожертвования на фиксированную сумму или оставляем поле для ввода суммы пользователю, далее при нажатии на кнопку нужно просто ввести данные банковской карты и все, платеж улетает вам.

**Comment by motienko** [comment]
Насколько я помню, совсем недавно там можно было донатить прямо с карты, без всяких акаунтов.

**Comment by CHERTS** [comment]
> Насколько я помню, совсем недавно там можно было донатить прямо с карты, без всяких акаунтов.

Да, действительно, с карты можно без регистрации. Зря я на Yandex булочку крошил. Извиняюсь.

**Comment by paulvi** [comment]
Olga wrote in https://github.com/Olga-Yakovleva/RHVoice/issues/75#issuecomment-403265488

> Хотя у моего синтезатора есть такие возможные прямые расходы: могло бы помочь платить за аренду сервера, на котором собираются голоса.

Ольга для программного решения с 100% окрытым кодом, не стоить платить за аренду сервера,
потому что есть много билд-сервером, которые предаставляют бесплатный сервис для окрыттых проектов.

Например https://travis-ci.org/, который легко интегрируется с github.
То есть весь вопрос в билд-скрипте, который бы полностью автоматически делал сборку и говотил релиз.
